src/api/Main/BuisnessLayer/Shrooms.Domain/Services/Wall/WallService.cs

```diff
@@ -752,23 +752,32 @@ private async Task<IEnumerable<WallDto>> GetAllOrNotFollowedWalls(UserAndOrganiz
 
         private async Task<IEnumerable<WallDto>> GetUserFollowedWalls(UserAndOrganizationDTO userOrg)
         {
-            var walls = await _wallUsersDbSet
-                .Include(x => x.Wall)
-                .Where(x => x.UserId == userOrg.UserId && x.Wall.OrganizationId == userOrg.OrganizationId)
-                .Where(x => x.Wall.Type == WallType.Main || x.Wall.Type == WallType.UserCreated)
```

Comment:
`x.Wall.Type == WallType.UserCreated` might be still needed, because it should reduce select amount significantly (my mentioned "select" query was just an example).
There are a lot of "dynamic" walls created, which are event related walls (WallType.Events) - i.e. when user joins the event, he joins such wall, and in this case these walls are not needed to be returned.